src/main/features/core/lyrics/index.ts

```diff
@@ -61,7 +69,12 @@ const getRemoteLyrics = async (song: QueueSong) => {
   let lyricsFromSource = null;
 
   for (const source of sources) {
-    const params = { artist: song.artistName, name: song.name };
+    const params = {
+      album: song.album || song.name,
```

Comment:
Is there a reason the album parameter is falling back to the song name instead of being left undefined if there's no album?

Reply:
Yes; lrclib requires you to provide an album parameter, and I suspect if there is no album name then it might be a single. I guess an alternative is to just make that assumption in the lrclib fetcher?

Reply:
Ok yeah it seems all the parameters are required in the search query. 
It's fine to leave as-is then.